backend/uclapi/workspaces/occupeye/cache.py

```diff
@@ -118,7 +118,7 @@ def cache_survey_data(self):
             staff_survey = str(
                 int(survey["SurveyID"]) in self._const.STAFF_SURVEY_IDS
             )
-
+            location_data = self._const.SURVEY_LOCATIONS[survey["Name"]]

```

Comment:
What if it doesn't exist? They add surveys without telling us.

Reply:
Added a null condition!